collectd/map.jinja

```diff
@@ -38,6 +41,7 @@
 {% set remote_client = salt['grains.filter_by']({
     'default': {
         'service': 'remote_collectd',
+        'defaults_file': '/etc/default/collectd',
```

Comment:
See https://github.com/tcpcloud/salt-formula-collectd/blob/master/collectd/files/collectd_systemd.service#L13. The file name is not the same for remote_collectd.